### PR TITLE
Deprecate PyPy3 installers

### PR DESCRIPTION
**Pull request by hmaarrfk**
Gonna leave this up here for 1 week for comments ahead of cutting a release:

Closes https://github.com/conda-forge/miniforge/issues/656

Checklist
* [ ] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [ ] Bumped the build number (if the version is unchanged)
* [ ] Reset the build number to `0` (if the version changed)
* [ ] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [ ] Ensured the license file is being packaged.

### CODE REVIEW

File: Miniforge3/pypy_deprecation.sh [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+    echo "Future Miniforge releases will NOT build with PyPy in the base environment installers."
+    echo "We advise you switch to Miniforge3 at your earliest convenience."
+    echo "More details at https://conda-forge.org/news/2024/08/14/sunsetting-pypy/"
+    echo "If you are unable to switch to Miniforge, you may pin your installer version to one found in "
+    echo "https://github.com/conda-forge/miniforge/releases/tag/24.3.0-1"
+    echo "or if you lack the system requirements (Linux glibc >= 2.17, or macOS + x86-64bit >= 10.13)"
+    echo "you may pin your installer to one older version found in "
+    echo "https://github.com/conda-forge/miniforge/releases/tag/24.3.0-0"
+    echo "This Miniforge installer will ceese to work in 2025."
+fi
```
Comment (jakirkham) [inline Miniforge3/pypy_deprecation.sh:16]: Let's put this in a text file that ships with the installer. Then all platforms can echo the test file and we can get the wording to match with both

**Review by jakirkham — COMMENTED** [review]
Thanks Mark! 🙏

Added a suggestion below

File: Miniforge3/pypy_deprecation.sh (hunk 1 above)
Comment (hmaarrfk) [reply Miniforge3/pypy_deprecation.sh:16]: I would rather not overbuild for a depreation that will only last 3 months.

The challenge is that the wording  MUST be different for different platforms.

Windows, linux, and OSX have different recommendations.

Generic recommendations REALLY confuse first time users.

Is there a word here that you are concerned with getting correct?

Comment (hmaarrfk) [reply Miniforge3/pypy_deprecation.sh:16]: My windows skills are also seriously lacking..... so I would rather not try something new....

My main goal for this week is to give time for somebody that might care for this installer to stepup, but not to create more organizational work. "2" notices is not enough to reorganize things IMO.

**Pull request merged**